mozdownload/scraper.py

```diff
@@ -823,6 +824,9 @@ def binary_regex(self):
 
     def build_filename(self, binary):
         """Return the proposed filename with extension for the binary."""
+        if not self.locale_build:
+            self.timestamp = self.builds[self.build_index]
+
```

Comment:
This would only add the `timestamp` for en-US builds, but localized builds still have the non-prefixed name. See the one test for tinderbox scraper which you didn't update.

So we definitely have to set `self.timestamp` inside of `get_build_info` after the call to `get_build_info_for_index`, which currently only gets executed for `en-US`.

Maybe we shouldn't care about localized builds given that we do not push those to archive.mozilla.org anymore. Those only exist in Taskcluster.

Reply:
Hi, could you please let me know whether you want to have this change. The last paragraph stands in strong contrast to the request for change in the paragraphs before...

Sorry, whimboo, it is not clear to me.

Reply:
I think lets leave as it is. We shouldn't care that much about l10n builds on archive.mozilla.org. No-one will be interested in those anyway. So yes, lets leave this code as is.